src/Tinker.php

```diff
@@ -67,6 +69,36 @@ protected function createShell(BufferedOutput $output): Shell
         return $shell;
     }
 
+    public function removeComments(string $code): string
+    {
+        $tokens = token_get_all("<?php\n".$code.'?>');
+        $cleanCode = '';
+        foreach ($tokens as $token) {
+            if (is_string($token)) {
+                // simple 1-character token
+                $cleanCode .= $token;
+            } else {
+                // token array
+                list($id, $text) = $token;
+
+                switch ($id) {
+                    case T_COMMENT:
```

Comment:
Instead of a `switch`statement, use a dedicated function with early returns

Reply:
I refactored everything to collections, will push everything soon